docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -395,11 +395,12 @@ Building with Mandrel requires a custom builder image parameter to be passed add
 ./mvnw package -Pnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-mandrel:{mandrel-flavor}
 ----
 
-Please note that the above command points to a floating tag for ease of use,
-and to make sure you get the newest Mandrel version supported by Quarkus.
-Consider using specific tags when going into production.
-Check https://quay.io/repository/quarkus/ubi-quarkus-mandrel?tab=tags[here] to find out about available tags.
-
+Please note that the above command points to a floating tag.
+It is highly recommended that use the floating tag,
```

Comment:
Or: 

```suggestion
It is highly recommended to use the floating tag,
```

Reply:
Yeah it's probably better than my suggestion.